src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -60,6 +60,15 @@ public void CurrentManagedThreadId_Idempotent()
             Assert.Equal(Environment.CurrentManagedThreadId, Environment.CurrentManagedThreadId);
         }
 
+        [Fact]
+        public void ProcessId_MatchesExpectedValue()
+        {
+            Assert.Equal(Environment.ProcessId, Environment.ProcessId);
```

Comment:
Are you just trying to make sure that the getter returns the same value when called twice?  If so, add a comment as it looks odd.

Reply:
nit: Maybe we should also assert that it is >0 ?

Reply:
Done